tests/main/lxd-mount-units/task.yaml

```diff
@@ -46,13 +52,13 @@ execute: |
     lxd.lxc restart ubuntu
 
     echo "Sanity check that mount overrides were generated inside the container"
-    lxd.lxc exec ubuntu -- find /var/run/systemd/generator/ -name container.conf | MATCH "/var/run/systemd/generator/snap-core18.*mount.d/container.conf"
+    lxd.lxc exec ubuntu -- find /var/run/systemd/generator/ -name container.conf | MATCH "/var/run/systemd/generator/snap-${core_snap}.*mount.d/container.conf"
     lxd.lxc exec ubuntu -- test -f /var/run/systemd/generator/snap.mount
 
-    echo "Sanity check that core18 snap is mounted correctly"
-    # Make sure core18 is mounted and readable for a regular user
-    retry -n 5 --wait 1 sh -c 'lxd.lxc exec ubuntu -- sudo --user ubuntu --login ls /snap/core18/current'
-    retry -n 5 --wait 1 sh -c 'lxd.lxc exec ubuntu -- sudo --user ubuntu --login mount | grep core18| grep allow_other'
+    echo "Sanity check that $core_snap snap is mounted correctly"
+    # Make sure $core_snap is mounted and readable for a regular user
```

Comment:
nitpick, $ does not expand in the comments, but maybe it's clear that this is about whatever is the value of $core_snap